strings/ion_laws.json

```diff
@@ -136,10 +134,10 @@
         "MASKED",
         "UNIDENTIFIED",
         "WEIRD",
-        "NAKED",
-        "NUDE",
-        "TWERKING",
         "SPOILING",
+		"NAKED",
+		"NUDE",
+		"TWERKING",
         "REDACTED",
```

Comment:
I don't think it'll actually make any difference code wise, but everything in the file currently is indented using spaces rather than tabs:
![image](https://user-images.githubusercontent.com/57483089/126278818-8f4a1717-6993-4df4-8c81-8e6b68ef8e7c.png)
Might be good to keep to that pattern just for consistency if nothing else.
```suggestion
        "SPOILING",
        "NAKED",
        "NUDE",
        "TWERKING",
        "REDACTED",
```